src/Umbraco.Web/Editors/ContentController.cs

```diff
@@ -392,6 +404,32 @@ private ContentItemDisplay GetEmpty(IContentType contentType, int parentId)
             return mapped;
         }
 
+        /// <summary>
+        /// Gets a collection of empty content items for all document types.
+        /// </summary>
+        /// <param name="contentTypeKeys"></param>
+        /// <param name="parentId"></param>
+        [OutgoingEditorModelEvent]
+        public IDictionary<Guid, ContentItemDisplay> GetEmptyByKeys([FromUri] Guid[] contentTypeKeys, [FromUri] int parentId)
+        {
+            var result = new Dictionary<Guid, ContentItemDisplay>();
+
+            using var scope = _scopeProvider.CreateScope(autoComplete: true);
+            var contentTypes = Services.ContentTypeService.GetAll(contentTypeKeys).ToList();
+
+            foreach (var contentType in contentTypes)
+            {
+                if (contentType is null)
```

Comment:
This whole block could just be:

```cs
var result = Services.ContentTypeService.GetAll(contentTypeKeys)
   .WhereNotNull()
   .ToDictionary(x => x.Key, x => GetEmpty(x, parentId));
```

Reply:
Ah good catch, however, I'm changing this method in the other PR, so I think I'll just merge this as-is for now, just to avoid a merge conflict 😄